DAO.sol

```diff
@@ -821,7 +822,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        // this can only be called after `quorumHalvingPeriod` has passed or at anytime
+        // by the curator with a delay of at least `minProposalDebatePeriod` between the calls
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now - minProposalDebatePeriod)) {
```

Comment:
The `minProposalDebatePeriod` is here to give token holders a chance to split before the quorum gets reduced? 

But the user has no indication of the curators intent to reduce the quorum until they call this function. 

Which can be call after just before a proposal that does not meet the current quorum is executed reducing the quorum and allowing it be executed. Allowing the quorum be reduced second before ETH gets transfered.